textproc/unicode/Portfile

```diff
@@ -40,7 +40,7 @@ destroot {
 }
 
 if {${name} eq ${subport}} {
-    revision            0
+    revision            1
```

Comment:
Ok, except the revision of the paracode subport also needs to be increased.

Reply:
good point - sorry I missed that one; corrected now